examples/simple-pipeline/simple_pipeline/config.yaml

```
@@ -0,0 +1,58 @@
+rago_pipeline: simple
```

```suggestion
rago_pipeline: simple_pipeline
```
minta tolong diupdate sesuai module name / foldernya

---

udpated kak